src/collectors/httpflat/httpflat.py

```diff
@@ -0,0 +1,59 @@
+# coding=utf-8
+
+"""
+Simple collector which get FLAT files and parse it into flat metrics
+
+#### Dependencies
+
+ * urllib2
+
+"""
+
+import urllib2
+import diamond.collector
+
+
+class HTTPFLATCollector(diamond.collector.Collector):
+
+    def get_default_config_help(self):
+        config_help = super(HTTPFLATCollector, self).get_default_config_help()
+        config_help.update({
+            'url': 'Full URL'
+        })
+        return config_help
+
+    def get_default_config(self):
+        default_config = super(HTTPFLATCollector, self).get_default_config()
+        default_config.update({
+            'path': 'httpflat',
+            'url': 'http://localhost/stat'
+        })
+        return default_config
+
+    def _flat_to_flat_metrics(self, prefix, data):
+        data = dict(line.split("=") for line in data.split("\n"))
+        for key in data:
+            yield key, data[key]
+
+    def collect(self):
+        url = self.config['url']
+
+        req = urllib2.Request(url)
+        req.add_header('Content-type', 'application/text')
```

Comment:
Also worth noting, there is no such thing as an `application/text` type. If anything, it'd be `text/plain`, but on a request, this describes the content-type of the body. In this case, you're making a GET, so there is no body, and this isn't needed.